### ISSUE TITLE
Wallet crashes on restructure branch if .cookie file exists

### ISSUE THREAD

**Issue by EagleEye25** [particl/partgui#467]
# Describe bug/issue

The wallet crashes if the .cookie file exists. It seems to be an authentication issue.
This only happens on the develop and restructure branch
`ThreadRPCServer incorrect password attempt from 127.0.0.1:1090`

# How to reproduce

Describe the steps needed to recreate the bug/issue:

1. Run wallet
2. Try to restart wallet

# More information

- Operating system and version: Windows 7
- Client version: Restructure

**Comment by pciavald** [comment]
should be fixed by #455. Please reopen if not

**Comment by rynomster** [comment]
Issue still exists. We can close this once the PR is merged

**Comment by vikas-cis** [comment]
Issue still exists on dev and I guess #455 has been merged in dev
ThreadRPCServer incorrect password attempt from 127.0.0.1:52.329
Windows 7

**Comment by pciavald** [comment]
duplicate of #392
